builder/js/ncf.js

```diff
@@ -1066,7 +1111,7 @@ $scope.onImportFileChange = function (fileEl) {
     return { "method_calls" : []
     , "name"         : ""
     , "description"  : ""
-    , "version"      : "1.0"
```

Comment:
it's not the same version ;) should still be 1.0! here it is the version of the technique not the import version